std/cs/_std/haxe/Exception.hx

```diff
@@ -77,27 +67,28 @@ class Exception extends NativeException {
 	function get_stack():CallStack {
 		return switch __exceptionStack {
 			case null:
-				var nativeTrace = new cs.system.diagnostics.StackTrace(native, true)
+				var nativeTrace = new cs.system.diagnostics.StackTrace(__nativeException, true);
 				__exceptionStack = CallStack.makeStack(nativeTrace);
 			case s: s;
 		}
 	}
 }
 
 @:dox(hide)
+@:nativeGen
 @:noCompletion
 @:native('System.Exception')
 private extern class NativeException {
 	@:noCompletion private function new(message:String, innerException:NativeException):Void;
-	@:noCompletion private final Data:cs.system.collections.IDictionary;
-	@:noCompletion private var HelpLink:String;
-	@:noCompletion private final InnerException:cs.system.Exception;
-	@:noCompletion private final Message:String;
-	@:noCompletion private var Source:String;
-	@:noCompletion private final StackTrace:String;
-	@:noCompletion private final TargetSite:cs.system.reflection.MethodBase;
-	@:overload @:noCompletion private function GetBaseException():cs.system.Exception;
-	@:overload @:noCompletion private function GetObjectData(incs.system.fo:SerializationInfo, context:StreamingContext):Void;
-	@:overload @:noCompletion private function GetType():cs.system.Type;
-	@:overload @:noCompletion private function ToString():cs.system.String;
+	@:noCompletion @:skipReflection private final Data:cs.system.collections.IDictionary;
```

Comment:
Can't this meta be applied to the whole class?

Reply:
meta.json says "TClassField" only. But I'll try

Reply:
Nope. Doesn't work on a class.